### PR TITLE
Improve and update docs

### PR DESCRIPTION
**Pull request by CanePlayz**
This pull request is aimed at improving and updating the outdated documentation. This is just a first draft to look at, more improvements will be made.

I've also decided to split the getting started guide into a basic, a GUI, and a CLI part to reduce the massive length of that file. Help with new options I missed or factual corrections on the more recent stuff would be much appreciated.

### CODE REVIEW

File: .docs/Dotnet.md [hunk 1]
```diff
@@ -9,44 +9,44 @@ If you are using the CLI from Docker, installing the runtime on the host machine
...
+- [.NET v7.0 Runtime for **Macs with Apple silicon**](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-7.0.11-macos-arm64-installer)
+- [.NET v7.0 Runtime for **Macs with an Intel processor**](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-7.0.11-macos-x64-installer)
 
 > **Note**:
-> [How can I tell if my computer is running a 32-bit (x86) or a 64-bit (x64) version of Windows?](https://support.microsoft.com/help/15056/windows-32-64-bit-faq)
+[How can I tell if my computer is running a 32-bit (x86) or a 64-bit (x64) version of Windows?](https://support.microsoft.com/help/15056/windows-32-64-bit-faq)
 
 > **Note**:
-> [Is my Mac using an Intel processor or Apple silicon?](https://support.apple.com/HT211814)
+[Is my Mac using an Intel processor or Apple silicon?](https://support.apple.com/HT211814)
```
Comment (Tyrrrz) [inline .docs/Dotnet.md:27]: The `>` should probably stay?

File: .docs/Docker.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Note the `-v /path/on/machine:/out` option, which instructs Docker to bind the `
 > **Note**:
 > If you are running SELinux, you will need to add the `:z` option after `/out`, e.g.:
 >
-> ```
+> ```powershell
```
Comment (Tyrrrz) [inline .docs/Docker.md:48]: Maybe we should use `console` for these too?

File: .docs/Using-the-GUI.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+### Step 2
+
+Please refer to the on-screen instructions to get your token, then paste your token in the upper text box and hit ENTER or click the arrow (→).
+
+> **Warning**:
+> **Never share your token!**
+> A token gives full access to an account, treat it like a password.
+
+<img src="https://i.imgur.com/SuLQ5tZ.png" height="400"/>
```
Comment (Tyrrrz) [inline .docs/Using-the-GUI.md:23]: Do we need to control the `height` here or can we get away with a normal markdown image markup?

File: .docs/Using-the-GUI.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+- **Reuse assets** (Optional) - If this option is set, the export will reuse already downloaded assets to skip redundant requests. This option is only available when **Download assets** is enabled.
+
+- **Assets directory path** (Optional) - If this option is set, the export will use the specified directory to store assets from all exported channels in the same place.
+
+> **Note**:
+> You need to scroll down to see all available options.
+
+## Settings
+
+**Auto-update**
```
Comment (Tyrrrz) [inline .docs/Using-the-GUI.md:64]: Should these be list items? I.e. start with `-`

**Review by Tyrrrz — COMMENTED** [review]
Nice, thank you 🙂

File: .docs/Dotnet.md (hunk 1 above)
Comment (CanePlayz) [reply .docs/Dotnet.md:27]: Yeah, I'll leave it. It doesn't make any difference though with how the blockquote is displayed.

File: .docs/Using-the-GUI.md (hunk 3 above)
Comment (CanePlayz) [reply .docs/Using-the-GUI.md:23]: It gets way too big with a normal markdown image.

File: .docs/Using-the-GUI.md (hunk 4 above)
Comment (CanePlayz) [reply .docs/Using-the-GUI.md:64]: Yep, good point.

File: .docs/Docker.md (hunk 2 above)
Comment (CanePlayz) [reply .docs/Docker.md:48]: Yup, I will change those.

File: .docs/Dotnet.md (hunk 1 above)
Comment (Tyrrrz) [reply .docs/Dotnet.md]: Easier to visually parse this way

File: .docs/Scheduling-MacOS.md [hunk 5]
```diff
@@ -200,19 +200,19 @@ Every Sunday at midnight and every Wednesday full hour (xx:00). Notice the inclu
...
-		<key>Minute</key>
-		<integer>00</integer>
-	</dict>
+ <dict>
+  <key>Weekday</key>
+  <integer>3</integer>
+  <key>Minute</key>
+  <integer>00</integer>
+ </dict>
 </array>
```
Comment (Tyrrrz) [inline .docs/Scheduling-MacOS.md:216]: Looks like indentation is messed up here. Can you please re-indent using 2 spaces for each level?

File: .docs/Scheduling-Windows.md [hunk 6]
```diff
@@ -38,9 +38,11 @@ exit
 - `dirhere` with the export directory (e.g. C:\Users\User\Documents\Exports)
 - `formathere` with one of the available export formats
 
-Make sure not to delete the quotes (")
+    Make sure not to delete the quotes (")
```
Comment (Tyrrrz) [inline .docs/Scheduling-Windows.md:41]: Why the 4 spaces here?

Comment (CanePlayz) [reply .docs/Scheduling-Windows.md:41]: Removed them, probably did that instinctively to put that line on the same indentation level as the stuff above it

**Comment by Tyrrrz** [comment]
Thank you. It looks good :) Ready to merge?

**Comment by CanePlayz** [comment]
Yup, I think I got everything in that I wanted

**Comment by Tyrrrz** [comment]
Merged, thank you

**Pull request merged**